internal/app/app.go

```diff
@@ -90,7 +93,7 @@ func NewIncidentRepository(ctx context.Context, logger log.Logger) model.Inciden
 // NewServiceRepository creates a new connection with the database for services
 func NewServiceRepository(ctx context.Context, logger log.Logger) model.ServiceRepository {
 	configuredDatabase := config.Env.Database
-	logger.Info(ctx, fmt.Sprintf(
+	logger.Debug(ctx, fmt.Sprintf(
```

Comment:
I would actually keep this one as Info, since it is just a one-line log, runs only once, in the beginning, and brings a relevant information (the configured service repository)